qiskit/aqua/operators/state_fns/circuit_state_fn.py

```diff
@@ -365,5 +373,5 @@ def permute(self, permutation: List[int]) -> 'CircuitStateFn':
         Returns:
             A new CircuitStateFn containing the permuted circuit.
         """
-        new_qc = QuantumCircuit(self.num_qubits).compose(self.primitive, qubits=permutation)
+        new_qc = QuantumCircuit(max(permutation) + 1).compose(self.primitive, qubits=permutation)
```

Comment:
This suggests that I can do 
```python
fn = One.to_circuit_op()  # CircuitStateFn
fn.permute([10000])
```
Is this the case? Is that also true for all operators?

Reply:
Yes you can do it for all operators.
Sorry I missed this comment earlier.